File: RepoTransBench/target_projects/Java/C#/code4craft_hello-design-pattern/tests/original/HelloWorldTest.cs
using Xunit;

namespace HelloDesignPattern.Tests
{
    public class HelloWorldTest
    {
        class Impl : IHelloWorld
        {
            public string HelloWorld() => "custom";
        }

        [Fact]
        public void TestHelloWorldInterfaceImpl()
        {
            IHelloWorld hw = new Impl();
            Assert.Equal("custom", hw.HelloWorld());
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/code4craft_hello-design-pattern/tests/original/hello_world_test.go
package tests

import (
	"testing"
)

type HelloWorld interface {
	HelloWorld() string
}

type implHelloWorld struct{}

func (i implHelloWorld) HelloWorld() string {
	return "custom"
}

func TestHelloWorldInterfaceImpl(t *testing.T) {
	var hw HelloWorld = implHelloWorld{}
	got := hw.HelloWorld()
	if got != "custom" {
		t.Errorf("Expected 'custom', got '%s'", got)
	}
}